src/mlbstandings/service.py

```
@@ -0,0 +1,37 @@
+        # 'worker_class': 'sync',
+        'threads': 8,
+        'timeout': 0,
+        # 'keepalive': 2,
+        # 'max_requests': 1000,
+        # 'max_requests_jitter': 100,
+        # 'logger_class': 'debug',  # ?
+    }
+
+    StandaloneApplication(app, options).run()
+
+def serve() -> None:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)